src/chatterbox/mtl_tts.py

```
@@ -0,0 +1,373 @@
+        else:
+            # Text is too large - split into chunks and process separately
+            #print(f"Text too large ({estimated_tokens} tokens), splitting into chunks...")
+            #print(f"Using cache-aware chunk limit: {max_chunk_tokens} tokens (cache_len={max_cache_len}, max_new={max_new_tokens})")
+            text_chunks = smart_text_splitter(text, max_chunk_tokens, self.tokenizer, language_id)
+            #print(f"Split into {len(text_chunks)} chunks")
+            
+            # Store original conditionals for reuse
+            original_conds = self.conds.clone()
+            
+            audio_chunks = []
+            for i, chunk in enumerate(text_chunks):
```

Ruff (0.13.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)